test/parser/next/private_methods.ts

```diff
@@ -192,7 +192,9 @@ describe('Next - Private methods', () => {
     'foo() { this.#m, (() => this)().#m }',
     'foo() { this.#m, (() => this)().#m }',
     'foo() { this.#m, (() => this)().#m }',
-    'method() { super.#x(); }'
+    'method() { super.#x(); }',
+    '#method() { super.x(); }',
+    '#method() { super.#x(); }'
```

Comment:
just to double confirm:
```js
#method(){ super(); }
```
should be invalid.

Reply:
I see. there is already a case to test it: https://github.com/meriyah/meriyah/blob/6de707a0efb3053767deaa36b1ed6979b0d3f873/test/parser/next/private_methods.ts#L13